### DISCUSSION TITLE
Be able to write dimensions in combinations

### DISCUSSION THREAD

**Discussion by sergiobm3** [PrestaShop/PrestaShop#40163] [Ideas]
#### Is your feature request related to a problem?

I need to be able to add the dimensions of the different combinations of my products for shipping.

#### Describe the solution you would like

        be able to write dimensions to combinations via web service.

#### Alternatives you have considered

There is some module to add this, but I can't use it through web service

**Comment by hibatallahAouadni** [comment]
Hello @sergiobm3 

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. 

If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know! 

Thank you
